internal/webserver/webserver.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/clastix/capsule-proxy/internal/options"
 	req "github.com/clastix/capsule-proxy/internal/request"
+	"github.com/gorilla/mux"
```

Comment:
Import statements are not in the correct order.

Third packages as `github.com/gorilla/mux` must be in the second section, resulting as this:

```go
import (
        // this packages are from the standard library, they are first class citizens
	"context"
	"crypto/tls"
	"encoding/json"
	"fmt"
	"net/http"
	"net/http/httputil"
	"strconv"
	"strings"
	"time"

        // third party packages, in the second block
	capsulev1alpha1 "github.com/clastix/capsule/api/v1alpha1"
	"github.com/clastix/capsule/pkg/utils"
	"github.com/gorilla/mux"
	v1 "k8s.io/api/authentication/v1"
	"k8s.io/apimachinery/pkg/labels"
	"k8s.io/apimachinery/pkg/selection"
	ctrl "sigs.k8s.io/controller-runtime"
	"sigs.k8s.io/controller-runtime/pkg/client"

        // third block, internal packages
	"github.com/clastix/capsule-proxy/internal/options"
	req "github.com/clastix/capsule-proxy/internal/request"
)
```

JFI you may notice `github.com/clastix/capsule/pkg/utils` in the second block and an eyebrow could rise: that's ok since it's from the `capsule` repository, it's not `capsule-proxy`, so it has to be treated as a third package since it's also imported using the Go modules.